prisma-data-platform/accelerate/nextjs/README.md

```diff
@@ -0,0 +1,53 @@
+# Accelerate Next.js Starter Project
```

Comment:
similar thought as for the Pulse Resend example: let's strive for consistency across examples; I think the Pulse starter has a nice structure that we can re-use in other examples